src/styles/content.scss

```diff
@@ -186,4 +186,36 @@
     background: #cfcfcf;
     border: 0;
   }
+
+  .tooltip {
```

Comment:
Could you move this [here](https://github.com/jekyll/jekyll-admin/blob/master/src/styles/metafields.scss#L228)?